app/models/ApiKey.ts

```diff
@@ -13,6 +13,10 @@ class ApiKey extends Model {
   @observable
   name: string;
 
+  @Field
+  @observable
+  expiryAt?: string;
```

Comment:
```suggestion
  expiresAt?: string;
```

Sorry to be a pain, can we rename this property throughout? There are a few other places the similar concept exists in the codebase and it's always "expiresAt"

Reply:
No worries :) `expiresAt` looks cleaner.